src/eduid_webapp/email/views.py

```diff
@@ -219,28 +221,41 @@ def verify_link(user):
         db = current_app.proofing_statedb
         state = db.get_state_by_eppn_and_email(proofing_user.eppn, email, raise_on_missing=False)
 
+        url = urlparse.urljoin(current_app.config['DASHBOARD_URL'], 'emails')
```

Comment:
I would be glad if we used eduid_common.api.utils.urlappend instead of urljoin so we don't get the problem with trailing slashes or not.